ForumProject/notifications/admin.py

```diff
@@ -15,3 +15,15 @@ def has_change_permission(self, request, obj=None):
 
     def has_delete_permission(self, request, obj=None):
         return False
+    
+@admin.register(NotificationPreferences)
+
+class NotificationPreferencesAdmin(admin.ModelAdmin):
+    list_display = ['id',
+        'startup',
+        'email_on_followers_change',
+        'email_on_share_subscription',
+        'in_app_on_followers_change',
+        'in_app_on_share_subscription'
+    ]
```

Comment:
The indentation and alignment of fields could be more consistent to improve code readability:
```
class NotificationPreferencesAdmin(admin.ModelAdmin):
    list_display = [
        'id',
        'startup',
        'email_on_followers_change',
        'email_on_share_subscription',
        'in_app_on_followers_change',
        'in_app_on_share_subscription'
    ]
```

Reply:
Done, thank you